Source/relay/TourGuide/Quests/Level 7.ash

```diff
@@ -92,10 +93,10 @@ void QLevel7GenerateTasks(ChecklistEntry [int] task_entries, ChecklistEntry [int
 		string text;
 		if (evilness == 0)
 			text = "Finished";
-		else if (evilness <= 25)
+		else if (evilness <= CYRPT_BOSS_EVILNESS)
```

Comment:
i am mildly annoyed by the <= here and < elsewhere but this should work so i do not care